### PR TITLE
Update Icons

### PR DESCRIPTION
**Pull request by crousselle**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @crousselle has signed the CLA. Thanks for the pull request!

File: BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m [hunk 1]
```diff
@@ -60,7 +60,7 @@ + (UIImage *)box_iconForFileName:(NSString *)fileName
 
 + (UIImage *)box_genericFileIcon
 {
-    return [UIImage box_iconWithName:@"icon-file-generic"];
+    return [UIImage box_iconWithName:@"other"];
```
Comment (adempsey) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: "other" is a little vague. acknowledging this is for the miscellaneous case, could we name it something a bit more detailed?

File: BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m [hunk 2]
```diff
@@ -94,37 +94,37 @@ + (NSString *)iconNameForFileExtension:(NSString *)fileExtension
...
-        iconName = @"Movie";
+        iconName = @"video";
     }
     else if ([[self docFileExtensions] containsObject:fileExtension]) {
-        iconName = @"Word";
+        iconName = @"word";
     }
     else if ([[self codeFileExtensions] containsObject:fileExtension]) {
-        iconName = @"Code";
+        iconName = @"cpde";
```
Comment (adempsey) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: typo - `cpde` not `code`

File: BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m [hunk 3]
```diff
@@ -94,37 +94,37 @@ + (NSString *)iconNameForFileExtension:(NSString *)fileExtension
...
     }
     else if ([[self textFileExtensions] containsObject:fileExtension]) {
-        iconName = @"Doc";
+        iconName = @"doc";
```
Comment (adempsey) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: would it be better for this to be named `text?`

File: BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m [hunk 4]
```diff
@@ -94,37 +94,37 @@ + (NSString *)iconNameForFileExtension:(NSString *)fileExtension
 {
     NSString *iconName = nil;
     if ([[self audioFileExtensions] containsObject:fileExtension])  {
-        iconName = @"Audio";
+        iconName = @"music";
```
Comment (adempsey) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: minor, I kinda like these as 'audio' and ' image' as opposed to 'music' and 'photo', at least on our side. but ok

**Pull request merged**